### PR TITLE
Fixed file extension when automatically searching for a subtitles and use original format is enabled

### PR DESCRIPTION
**Pull request by morpheus65535**

### CODE REVIEW

File: bazarr/subtitles/manual.py [hunk 1]
```diff
@@ -158,7 +158,7 @@ def manual_download_subtitle(path, audio_language, hi, forced, subtitle, provide
         subtitle.language.forced = True
     else:
         subtitle.language.forced = False
-    if use_original_format in ("1", "True"):
+    if use_original_format in (1, "1", "True", True):
```
Comment (anderson-oki) [inline bazarr/subtitles/manual.py:161]: Wouldn't `bool(use_original_format)` be better?

Comment (anderson-oki) [reply bazarr/subtitles/manual.py:161]:
```suggestion
    if bool(use_original_format):
```

Comment (anderson-oki) [reply bazarr/subtitles/manual.py:161]: Nvm, there are false results.

File: custom_libs/subliminal_patch/subtitle.py
Comment (JaiZed) [inline custom_libs/subliminal_patch/subtitle.py:1]: What is the purpose of keeping the commented out lines 319-322?

Comment (morpheus65535) [reply custom_libs/subliminal_patch/subtitle.py:1]: None... they were there and I didn't changed it.

Comment (JaiZed) [reply custom_libs/subliminal_patch/subtitle.py:1]: Except for one line, they are duplicates of the lines immediately above.
Obviously there for temporary debugging.
You don’t want to clean up after @vitiko98 ? :)

**Review by JaiZed — APPROVED** [review]
It all looks good to me.
A big improvement from before.
Ship it!

**Pull request merged**